lib/client.js

```diff
@@ -215,6 +215,12 @@ function processResponse(path, result) {
   return result;
 }
 
+function setupTracking(client) {
+  window.addEventListener('click', function() {
+    client.invoke('track', { type: 'click' }) // Arturo filtering is being done on the ZAF side
```

Comment:
semicolon 👴

Reply:
I'm used to a linter autofixing that for me 😆